server/routes/user_screen/fetchById.js

```diff
@@ -0,0 +1,16 @@
+const userScreen = require("../../query/user_screen");
+const express = require("express");
+const router = express.Router();
+
+router.get("/:id", async (req, res) => {
+  try {
+    const id = req.params.id;
+    const response = await userScreen.fetchById(id);
+    res.json(response);
+  } catch (error) {
+    res.json(error);
+    console.error("GET user_screen: ", error);
```

Comment:
should this error message specify that it's fetching by id?

Reply:
i think in the future we could definitely come up with a better way of doing error messages, like having a standard that applies to every function so they're all consistent